code/modules/antagonists/revolutionary/team_revolution.dm

```diff
@@ -62,6 +63,17 @@
 			return FALSE
 
 	SSshuttle.clearHostileEnvironment(src) // revs can take the shuttle too if they want
+
+	var/msg = "The revolutionaries have won! All command, security, and legal jobs have been removed. You can change this with the \"Free Job Slot\" verb."
```

Comment:
this seems directed at admins during the round, could you create a second msg that is better suited for adding logs. I.e. something like "Revolutionary win conditions met, command, security, and legal jobs are now closed."